src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.VSCodeWorkspaces/plugin.json

```diff
@@ -0,0 +1,12 @@
+{
+  "ID": "CEA0FDFC6D3B4085823D60DC76F28100",
+  "ActionKeyword": "*",
```

Comment:
This is such a special plugin that imo a keyword make sense.

Reply:
Agreed. This plugin needs its own activation key and it should not be included in the default search.
In 0.33 there will be the plugin manager to change this behavior but for now it cannot be executed by default.

Reply:
@enricogior
Do you want to merge for 0.31? Don't think that we get it ready for 0.31. When do we feature close 0.31?

Reply:
If it's disabled by default it's OK to merge it for 0.31

Reply:
> * Can we use the result images without getting problem/trouble because of copyright?
 I forgot about this. I will do some changes.
> * What happened if the plugin is triggered and no vscode is installed or no workspaces existing? I this case catched? We should not get crashes in this case.
If vscode is not installed or no workspace exists the plugin return an empty list.

Reply:
i already added an ActionKeyword and the made the necessary chages